uploader.json

```diff
@@ -3355,6 +3355,11 @@
             "symbolic_name": "kubepods_cpu_quota",
             "pattern": []
         },
+	{
+            "glob": "/var/log/libvirt/qemu/*.log",
+	    "pattern": [],
+	    "symbolic_name": "libvirtd_qemu_log"
```

Comment:
@psachin please fix the indentation here and also add a spec to the files section for the V1 clients to this file (uploader.json) only:
```
    "file": "/var/log/libvirt/qemu/().*\\.log"
```